### PR TITLE
#142 float images

### LINKED ISSUE #142: Small Float images
from Josh: 

> 6. We had some discussion about images that display within the article copy. The decision was made to move them on their own “line”. Although i prefer the copy wraps the images, i understand this is more manual/complex to integrate. However, on some pages it just doesn’t translate as well, for example on this page we now have a huge image that pushes all the content down: https://main--24life--hlxsites.hlx.page/fitness/2018/a-personal-trainer-on-getting-race-day-ready-and-total-body-cardio-moves - thoughts on this?

Idea: 
add a section metadata style which makes the images float left or right. Apply to some of the already imported articles. Like `float-images-left`, `float-images-right`, `small-images`.

### PR DESCRIPTION
**Pull request by synox**
I updated a handful of articles where floats is better. 

Fix #142 

Test URLs:
- Before: https://main--24life--hlxsites.hlx.page/fitness/2018/a-personal-trainer-on-getting-race-day-ready-and-total-body-cardio-moves
- After: https://142-float--24life--hlxsites.hlx.page/fitness/2018/a-personal-trainer-on-getting-race-day-ready-and-total-body-cardio-moves

- https://142-float--24life--hlxsites.hlx.page/focus/2016/pack-your-bag
- https://142-float--24life--hlxsites.hlx.page/focus/2017/four-reasons-to-meet-up-to-meditate
- https://142-float--24life--hlxsites.hlx.page/focus/2020/wake-up-work-out-get-well-best-of-ces-2020

Library:
- After: https://142-float--24life--hlxsites.hlx.page/block-library/blocks/float-images

### CODE REVIEW

File: scripts/scripts.js [hunk 1]
```diff
@@ -14,6 +14,35 @@ import {
...
+    if (picture.closest('div.block')) {
+      // don't link if already in a block.
+      // eslint-disable-next-line no-continue
+      continue;
+    }
+
+    const link = document.createElement('a');
+    link.href = picture.querySelector('img').src;
+    link.innerHTML = picture.outerHTML;
+    picture.replaceWith(link);
```
Comment (iamitshri) [inline scripts/scripts.js:42]: Was't the picture a required tag for the browser to choose the best image?

File: blocks/footer/footer.css [hunk 2]
```diff
@@ -4,6 +4,7 @@ footer {
   background-color: var(--footer-background-color);
   font-size: var(--footer-font-size-copyright);
   color: var(--text-color-on-black-background);
+  clear: both;
```
Comment (iamitshri) [inline blocks/footer/footer.css:7]: what does this do?

Comment (synox) [reply blocks/footer/footer.css:7]: Prevents floating images to enter this area. 
https://developer.mozilla.org/en-US/docs/Web/CSS/clear

Comment (synox) [reply blocks/footer/footer.css:7]: without clear:
Screenshot 2023-09-11 at 09 30 49

Comment (iamitshri) [reply blocks/footer/footer.css:7]: Thanks for the image 👍🏽

**Pull request merged**